project-reports/2024/2024-Q2-Hyperledger-Identus.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: 2024 Q2 Hyperledger Identus
+parent: 2024
+grand_parent: Project Updates
+nav_exclude: true
+---
+
+Created by Fayyaadh Adams with input from other maintainers
+
+
+# Project Health
+
+The project is fairly new to the Hyperledger Ecosystem however we continue to have steady contributors. We are currently not on the LFX dashboard yet and are waiting to be added to gain further insights. In the last three weeks alone, we have seen 3 new contributors (not code related) show up and additional usage of our code within production environments (https://socious.io/ - production use case). 
+
+
+# Questions/Issues for the TOC
+
+Is there a way to access funds for open source contirbutions and development? 
```

Comment:
```suggestion
Is there a way to access funds for open source contributions and development? 
```

Reply:
This is what is currently documented: https://wiki.hyperledger.org/display/HYP/Funding+Opportunities+for+Projects. It would be great to find out more about what you are looking for.

Reply:
Thank you for sharing. We did some polls in the community and it seems there is a tendency for community members and potential contributors to participate more if there is funding. It's something we might explore and test the waters with to see if that attracts more contributors and maintainers